package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "gh-node-module-generatebom",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Generate a CycloneDX SBOM for Node.js",
+  "author": "Patrick Dwyer",
+  "license": "Apache-2.0",
+  "main": "lib/main.js",
+  "homepage": "https://github.com/CycloneDX/gh-node-module-generatebom#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/CycloneDX/gh-node-module-generatebom"
+  },
+  "bugs": {
+    "url": "https://github.com/CycloneDX/gh-node-module-generatebom/issues"
+  },
+  "scripts": {
+    "build:bundle": "node ./esbuild.config.mjs",
+    "format:check": "prettier --check **/*.js",
+    "format:write": "npm run format:check -- --write"
+  },
+  "dependencies": {
+    "@actions/core": "^1.7.0"
+  },
+  "devDependencies": {
+    "chalk": "^5.0.1",
+    "esbuild": "^0.14.38",
```

Comment:
could you pin the build engine to an exact version?
this might help with `dependabot` integration in the future

Reply:
The `esbuild` is essential to the produced `dist`, and you intriduced tests for even that (<3) .

I want the dependency version locked to an exact version here, so that i can drive the modifications that `dependabot`  or `renovate` do in a way that is  decoupled from ordinary lockfile maintenance. 
While ordinary lockfile maintenance will just bump all the dependencies that have floating versions, this one should not be bumped in there, but should get an own PR if the version changes.